package.json

```diff
@@ -53,6 +54,7 @@
     "webpack": "2.7.0",
     "webpack-bundle-tracker": "0.2.1",
     "webpack-merge": "4.1.1",
+    "whatwg-fetch": "^2.0.3",
```

Comment:
It looks like the pattern is to pin these to an exact version (not the lack of `^`).

Reply:
sure, I just ran a command the linter told me to run. Should I just cut the ^?

Reply:
[check out the `--save-exact` `npm install` option](https://docs.npmjs.com/cli/install)

![image](https://user-images.githubusercontent.com/8136030/37120792-5df37a36-2229-11e8-91a4-ce8d13ec69a1.png)